package.json

```diff
@@ -33,9 +33,9 @@
   "devDependencies": {
     "@types/jest": "^27.0.2",
     "@types/markdown-table": "^3.0.0",
-    "@types/node": "^16",
+    "@types/node": "^20",
     "@typescript-eslint/parser": "^5.16.0",
-    "@zeit/ncc": "^0.22.3",
+    "@vercel/ncc": "^0.38.1",
```

Comment:
ncc build is failed.
https://github.com/aki77/simplecov-report-action/actions/runs/7824831804/job/21356571624

so, I use `vercel/ncc` as a refarence 
https://github.com/actions/typescript-action/blob/d88c8226076df14de5a25dfeb0a4961ad04a9084/package.json#L76